CarpetX/src/schedule.cxx

```diff
@@ -1112,6 +1130,14 @@ int Initialise(tFleshConfig *config) {
     CCTK_Traverse(cctkGH, "CCTK_RECOVER_VARIABLES");
     CCTK_Traverse(cctkGH, "CCTK_POST_RECOVER_VARIABLES");
 
+    // Here we assume that all levels have catched up to the coarsed one when
```

Comment:
"caught" I think. "coarsed" -> "coarse" or "coarsest" (not sure which one is intended, I suspect "coarsest").